es-models/gdc_from_graph/case.mapping.yaml

```diff
@@ -48,10 +48,18 @@ properties:
     type: keyword
   days_to_index:
     type: long
+  days_to_lost_to_followup:
+    type: long
   demographic:
     properties:
+      cause_of_death:
+        type: keyword
       created_datetime:
         type: keyword
+      days_to_birth:
+        type: long
+      days_to_death:
```

Comment:
OK, I remember this was discussed. There will be period of time 'mix' old and new will coexist. As brought up earlier, ES indexes built with data in this 'mix' state may not be able to drive the Data Portal in a desirable way. We probably don't need to worry about this now.